.gitignore

```diff
@@ -25,8 +25,7 @@ build/Release
 
 # Dependency directory
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
-node_modules
-app/node_modules
+/node_modules
```

Comment:
Forcing `node_modules` to only be ignored in the root feels like too strict of a config. Thoughts on that? Wht about changing this should be changed to `node_modules`?

Reply:
@amilajack That was a response to this comment from @vikr01: https://github.com/electron-react-boilerplate/electron-react-boilerplate/pull/1854#issuecomment-434074811

I could go either way on this. I can't think of any use case where a user *would* have node_modules in a subdirectory, but maybe?

Reply:
Yeah, I was thinking the same thing (conversation on [#1854](https://github.com/electron-react-boilerplate/electron-react-boilerplate/pull/1854#issuecomment-434074811)).

This actually could cause people using `electron-react-boilerplate` with subpackages to all of a sudden see their `node_modules` be included in unstaged files.

Reply:
Yeah, good point. Sounds like I should change it back to just node_modules... incoming...